### ISSUE TITLE
invalid mets.xml file after `ocrd workspace init`

### ISSUE THREAD

**Issue by finkf** [OCR-D/core#152]
The [test](https://github.com/cisocrgroup/cis-ocrd-py/blob/dev/tests/align/run_align_test.bash) for the aligner fails with the latest version of ocrd. 
It seems that the generated mets.xml is broken:

```xml
<?xml version="1.0" encoding="UTF-8"?>
b'<mets:mets xmlns:mets="http://www.loc.gov/METS/" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="info:lc/xmlns/premis-v2 http://www.loc.gov/standards/premis/v2/premis-v2-0.xsd http://www.loc.gov/mods/v3 http://www.loc.gov/standards/mods/v3/mods-3-6.xsd http://www.loc.gov/METS/ http://www.loc.gov/standards/mets/version17/mets.v1-7.xsd http://www.loc.gov/mix/v10 http://www.loc.gov/standards/mix/mix10/mix10.xsd">\n  <mets:metsHdr CREATEDATE="2017-11-30T16:18:26">\n    <mets:agent OTHERTYPE="SOFTWARE" ROLE="CREATOR" TYPE="OTHER">\n      <mets:name>DFG-Koordinierungsprojekt zur Weiterentwicklung von Verfahren der Optical Character Recognition (OCR-D)</mets:name>\n      <mets:note>OCR-D</mets:note>\n    </mets:agent>\n  </mets:metsHdr>\n  <mets:dmdSec ID="DMDLOG_0001">\n    <mets:mdWrap MDTYPE="MODS">\n      <mets:xmlData>\n        <mods:mods xmlns:mods="http://www.loc.gov/mods/v3">\n        </mods:mods>\n      </mets:xmlData>\n    </mets:mdWrap>\n  </mets:dmdSec>\n  <mets:amdSec ID="AMD">\n  </mets:amdSec>\n  <mets:fileSec>\n  </mets:fileSec>\n</mets:mets>\n'```

**Comment by finkf** [comment]
If I set 
```python
"install_requires"= [
 'ocrd <= 0.7.0',
]```
 the tests succeed again

**Comment by kba** [comment]
Apparently https://github.com/OCR-D/core/pull/151 broke this. I'll investigate.

**Comment by kba** [comment]
Should be fixed in 0.7.2.
